### ISSUE TITLE
Divine shield and Blessing of protection should prevent drown damage while active.

### ISSUE THREAD

**Issue by Mostprovidedlinks** [elysium-project/server#2429]
divine shield and blessing of protection prevent fall damage and physical damage and drowning is physical

http://wowwiki.wikia.com/wiki/Divine_Shield

On anathema either of them dont prevent drown damage

**Comment by Phoosy** [comment]
How about a citation that isn't reflective of the existing Divine Shield from _retail_ and instead of the iteration that is Vanilla-era mechanics?

**Comment by Mostprovidedlinks** [comment]
Vanilla text states all damage phoosy

**Comment by Mostprovidedlinks** [comment]
http://et.worldofwarcraft.wikia.com/wiki/Divine_Shield

Show the patch notes of all the patches say nothing about changing it to prevent drown damage. Meaning it should of always been that way.

**Comment by Phoosy** [comment]
The text may indicate all damage but there are notable exceptions to stated text in spells and abilities that make the plaintext unreliable. You need an actual citation demonstrating Paladins using the bubble to avoid taking damage during vanilla prior to ~December of 2006.

**Comment by Mostprovidedlinks** [comment]
The patch notes are the citation. They list the effects changed of the spell.
Since the divine shield had not had any changes in patch notes saying it prevents drowning and since it mentions it does outside of the patch note in the description of immunity that means it was originally that way from the start of the game. Otherwise you would see for example

Patch xxx.. 200... : Divine shield now affects drowning

Since you don't that proves It has been this way originally

**Comment by Mostprovidedlinks** [comment]
All damage means all damage.

**Comment by Mostprovidedlinks** [comment]
I can argue thesame thing and say find me proof in the patch notes saying it didn't originally in vanilla.
I think things shouldn't be fixed unless broken. Not fixed while it was originally fixed which creates a problem.

**Comment by Xaph-er** [comment]
You can't win against people who hate the class.

**Comment by Mostprovidedlinks** [comment]
http://db.vanillagaming.org/?spell=5697

Is a spell that prevents drowning.

Divine is also a spell that prevents drowning.

**Comment by gearselysium** [comment]
It doesn't prove anything. It merely suggests. Divine Shield is an immunity based spell. It doesn't apply Water Breathing. 

Divine Shield applies two different immunities - Physical, and All Spells. Drowning is considered environmental damage. As it currently stands, the only thing that prevents drowning is water breathing spell aura, OR having time left on your breath timer. 

See if you can find out if Ice Block prevented drowning in Vanilla 1.12.1

**Comment by gearselysium** [comment]
Anything that prevents physical damage also prevents falling damage - absorb or immunity.

**Comment by connormcgregor** [comment]
They already stated it prevents drown damage on that page and the patch notes show no changes to it.

That is viable proof

**Comment by gearselysium** [comment]
It's not proof. Many abilities have had incorrect or misleading tooltips and descriptions that differed to the actual implementation. 

There is no special handling for any immunity - no matter the source - to drowning in the core at all.

If you can find a video of a player being immune to drowning while bubble is up, or ice block, then that is primary evidence. We simply cannot trust web pages due to inaccuracies and incompleteness.

**Comment by Mostprovidedlinks** [comment]
The fact that a spell affects drowning such as underwater breathing proves that a spell like divine shield affects it. Patch notes prove the spell never changed.

**Comment by Mostprovidedlinks** [comment]
Its immunity based spell affects the drowning damage as its still damage.

**Comment by Mostprovidedlinks** [comment]
I cant find any proof telling me anywhere that divine shield cant prevent all damage.

**Comment by gearselysium** [comment]
You have a broken understanding of what "proves" means. Underwater Breathing is specifically tested for in the core. Underwater Breathing is a specific spell aura. You can see this applied by Unending Breath for example, from the datamined DBC information:

Effect #1 | (6) Apply Aura #82: Water Breathing Value: 0

The datamined DBC information for Divine Shield makes no reference to applying any kind of spell aura:


Apply Aura #39: Immune School (1) Value: 0  

- This applies immunity to PHYSICAL/NORMAL  

Apply Aura #138: Haste - Melee Value: -100

- This applies a negative melee haste effect 

Apply Aura #39: Immune School (126)

- This applies immunity to all spell schools (126 is the integer mask combining all spell schools)

There is no application of any other mod or aura.

**Comment by gearselysium** [comment]
In order for an immunity to prevent a damage type, the damage must be of that type. Drowning is not damage being applied to the character like slime, fire, or lava, or falling. Drowning cannot be absorbed by PWS for example, despite that PWS applies absorb for all damage (127 = 126 + 1)

Ice Block immunities are exactly the same as Divine Shield, so if Divine Shield prevents drowning, then Ice Block should also prevent drowning. 

Blessing of Protection on the other hand only applies immunity to normal/physical (1), and the definition of the spell specifically says "Attacks". It also does not prevent drowning.

.

**Comment by Mostprovidedlinks** [comment]
The problem is the damage from drowning is not an aura. Its damage

And divine shield does debuff auras from items in vanilla. Its already happening with a few items on elysium.

**Comment by gearselysium** [comment]
> The problem is the damage from drowning is not an aura. Its damage and its school would be affected by

This is not true. Damage applied from drowning is environmental and applied directly through the mirror timer calculations. It is not applied through spell schools or other forms of damage. The difference between fall damage (which is handled in the movement handler for falling) in this regard is that fall damage is specifically treated as physical damage. 

That is to say that drowning damage does not have a damage school. But even if it did, there are no immunity tests involved.

**Comment by gearselysium** [comment]
The point is not the Divine Shield doesn't prevent drowning, but that there needs to be substantial proof to add a specific change to the drowning handling that only targets Divine Shield and not any other identically blocking aura such as Ice Block, BoP, PWS etc - particularly if you're suggesting that the reason it prevents drowning is because drowning is a physical school damage calculation.

**Comment by Mostprovidedlinks** [comment]
https://wow.gamepedia.com/Ice_Block

Ice Block will prevent drowning and fatigue damage, and reset the damage counter (not the bar) to zero. You will still lose breath and become fatigued while Ice Blocked.

**Comment by Mostprovidedlinks** [comment]
Under no where does it say it has been added to prevent drowning.

Patches and hotfixes
Hotfix (2016-12-06): "Ice Block's cooldown will correctly reset at the end of an encounter."
Legion Patch 7.0.3 (2016-07-19): Now learned at level 26.
WoD Patch 6.0.2 (2014-10-14): Now learned at level 15.
MoP Patch 5.0.4 (2012-08-28): Now learned at level 26.
TBC Patch 2.3.2 (2008-01-08): Ice Block is trainable for mages of all talent builds at level 30.
TBC Patch 2.3.0 (2007-11-13): In Patch 2.3.0, Ice Block was reverted back to its old behavior.
Originally, a player who Ice Blocked in midair would continue to fall normally. At one point this was changed so an Ice Blocked player hovered while the spell was active, then began to fall when it expired.
TBC Patch 2.1.0 (2007-05-22): Hypothermia debuff added.
Prior to this, a mage with Ice Block and  [Cold Snap] could be protected for 20 consecutive seconds, with only a short interruption to use Cold Snap.
WoW Icon 16x16.png Patch 1.12.0 (2006-08-22): Fixed a bug that was causing Ice Block to trigger a 5 minute cooldown on the  [Gnomish Cloaking Device].
WoW Icon 16x16.png Patch 1.4.0 (2005-05-05): Fixed race condition where damage would sometimes be applied after Ice Block was activated.
Patch 0.9 (2004-08-17):
Duration slightly increased.
Now a buff instead of a debuff (you can cancel it).

**Comment by Mostprovidedlinks** [comment]
Effect #3 | Apply Aura: Immunity (All)

It implies this on every site

**Comment by Mostprovidedlinks** [comment]
Here I found it for you mate. 

http://www.wowhead.com/spell=63148/divine-shield

**Comment by gearselysium** [comment]
Again though, you're quoting from the "Tips and Tactics" section. It has never been part of the spell descriptions for either IB or DS that they prevent drowning.

> Divine Shield prevents all damage, including (unlike most other bubbles of other classes) falling damage, and even drowning damage. It is not uncommon for a Paladin to leap off a high cliff or despawn their flying mount in the air and bubble before landing, preventing damage or death. Hand of Protection does exactly the same thing for falling damage, and a Paladin with good timing may bubble himself and a party member.

The problem here is that HoP/BoP does not prevent drowning according to the same text that says DS does, but they both apply the same Physical immunity. PWS does not prevent drowning but applies absorb ALL damage. Ice Block and DS are essentially the same spell.

DI doesn't say it prevents drowning, but applies the same immunities.

**Comment by gearselysium** [comment]
> Here I found it for you mate.

You just linked a patch 7.0.1 version of a spell that I broke down for you earlier. 
https://github.com/elysium-project/server/issues/2429#issuecomment-327988251

**Comment by gearselysium** [comment]
Again, you're asking for a major core change, and your evidence is two comments in Tips and Tricks on pages that post date anywhere in the realm of time that we would be considering, and your basis out of this is that the spell hasn't changed - despite the fact that at no point in time has either spell officially been announced or described to stop drowning.

**Comment by Mostprovidedlinks** [comment]
That version is http://www.wowhead.com/spell=63148/divine-shield
is the Before Build 13875 divine shield which is the vanilla divine shield

Before Build 13875 is before catalcysm

as you can see on wowhead 
Protects the paladin from all damage and spells for 12 sec, but increases the time between your attacks by 100%. Once protected, the target cannot be made invulnerable by Divine Shield, Divine Protection or Blessing of Protection again
Before Build 13875

**Comment by Mostprovidedlinks** [comment]
Now look at the effects, they are identical to the version of the divine shield on cataclysm and beyonds divine shield
http://wowwiki.wikia.com/wiki/Divine_Shield
same icon as that divine shield and its database shows thesame effects.

They both remove auras and all damage

**Comment by Mostprovidedlinks** [comment]
wowhead is saying the original bubble on their page has aura immunity. You cant deny that mate.

**Comment by gearselysium** [comment]
The fact that DS removes auras has nothing to do with DROWNING. Drowning is an environmental effect, not an aura. None of these pages show any official statement that these spells prevent drowning and always have. 

http://et.worldofwarcraft.wikia.com/wiki/Breath

Makes no mention of DS or IB or any other immunity mechanic to mitigate drowning. All the items and abilties listed that are pertinent to Vanilla use the Water Breathing spell aura.

**Comment by gearselysium** [comment]
Look, this is how drowning works.

Every tick, the player update is processed. This checks whether the player is underwater, and if so, whether they were underwater previously. If you are underwater a check is made for the Water Breathing aura, if this isn't present and you aren't dead, the timer is shown using your maximum breath or updated using time since last tick. If your breath timer is shown and below 0, you take drowning damage. If your health drops below 0, you die.

That's the way the core handles drowning. There is no immunity mechanics involved, no absorbs. Nada. The only thing checked is whether you have Water Breathing. Even Druid's Aquatic From uses Water Breathing as the mechanic.

**Comment by Mostprovidedlinks** [comment]
Yes but if youre claiming pedia information and pedia states on another page that ds or ib does prevent drown damage. How can you use pedia  as a souce and state wowpedia isnt claiming ds prevents drown damage.

**Comment by gearselysium** [comment]
Exactly right. The information provided is indicative only, not conclusive.

**Comment by Mostprovidedlinks** [comment]
Problem is the 3rd effect of divine shield prevents all damage.
No matter what it is. It says this on wowhead.

All damage.

**Comment by Mostprovidedlinks** [comment]
The difference between divine shield and underwater breathing is underwater breathing removes the bar. Divine shield prevents the damage.

**Comment by Mostprovidedlinks** [comment]
Those pages are enough proof to have it fixed.

**Comment by gearselysium** [comment]
Drowning isn't technically damage, it's an environmental effect. You're not being hurt by the water, you're asphyxiating. Nothing else in the game has any mention of mitigating in any way the health ticks (which are based of % max hp) - there's no indication of any kind of mechanic through which drowning is mitigated. 

> The difference between divine shield and underwater breathing is underwater breathing removes the bar. Divine shield prevents the damage.

Then every aura that mitigates damage the same way DS does should mitigate drowning. DS only makes you immune to 127, same as IB and DI. PWS places a 127 absorb. 

 > Those pages are enough proof to have it fixed.

No, they are not. They are inconclusive at best.

**Comment by Mostprovidedlinks** [comment]
If it says it absorbs all damage then you cant deny it. Its 3rd effect must reset the bar and you must be immune to drowning. It says that on wowpedoa with iceblock.
The bar is reset.

And wowhead shows the old bubble has identical effects to the newer one stated on wowpedia to be immune to drowning damage. Meaning the 3rd bubble effect has the cancel drowning effect.

**Comment by Mostprovidedlinks** [comment]
Wowhead showing the vanilla bubble identical to the new bubble is more than enough.
Patch notes have no notes saying drowning was added.
Many pages state drowning cant happen with divine shield.

**Comment by gearselysium** [comment]
> If it says it absorbs all damage then you cant deny it. 

Again, just because it says one thing doesn't mean that's how it was implemented. PLEASE PROVIDE ACTUAL PROOF INSTEAD OF SOMEONE'S 2016 COMMENTS ABOUT IT FROM A PLAYER CREATED WEBSITE.

This means providing a video of it being used in retail vanilla in that way, or providing any such primary evidence.

Continuing to stick your fingers in your ear and chant "la la la i cant hear you it prevents drowning i dont have to prove it" wont get a change made.

> Wowhead showing the vanilla bubble identical to the new bubble is more than enough.

No it is not. Bubble has never officially prevented downing. Wowhead is a player created site not an official statement from Blizzard or an actual demonstration. 

> Patch notes have no notes saying drowning was added.

Patch notes have notes saying drowning mitigation ever existed. The ability has never been stated to prevent drowning so why would there be patch notes saying that it no longer does?\

> Many pages state drowning cant happen with divine shield.

These are all copies of the same page, and they are all player created, and the only places that say drowning can't happen with DS and IB are in the Tips and Tricks section and every single one of these pages was created or updated way past Vanilla. 

The information is NOT reliable. You need to prove that this was ever a thing, not just that some player one time said it. You can try using the wayback machine internet archive with a search date of 2005/6. That would help your cause, but continually reposting the same useless information over and over and then claiming you've proved it IS NOT PROOF.

If you do not prove that there is an issue and that it needs to be changed, it won't be. The burden is on you.

**Comment by Mostprovidedlinks** [comment]
Wowhead has proven it in the effects

**Comment by Mostprovidedlinks** [comment]
Wowheads vanilla retail bubble is identical

**Comment by gearselysium** [comment]
> Wowhead has proven it in the effects

No it hasn't. There are many abilities with the same immunity, some triggered by items, others added in different expansions, and these have never been indicated to mitigate drowning either. 

Drowning is not damage - it has no spell school, and there is no spell school flag that applies to it. 127 means immunity to all spells and normal damage. It doesn't mean immune to drowning. 

> Wowheads vanilla retail bubble is identical

So? This is irrelevant.

**Comment by Mostprovidedlinks** [comment]
If an items spll can affect drowning divine ahield can too

**Comment by Mostprovidedlinks** [comment]
Wrong again. Wowhead does have the retail vanilla bubble in the database. I have linked it for you 
The spell is word for word and identical to vanilla
It is the vanilla divine shield

Before Build 13875 

as you can see on wowhead
Protects the paladin from all damage and spells for 12 sec, but increases the time between your attacks by 100%. Once protected, the target cannot be made invulnerable by Divine Shield, Divine Protection or Blessing of Protection again
Before Build 13875

**Comment by Mostprovidedlinks** [comment]
Yes to all spells regardless of if drowning is damage related or not.

Yeah lalala lets disregard all of the information from wowhead.
Prevents all damage fron spells yet drowning can bypass. Do you have any idea how stupid this sounds?

Drowning is an environmental spell affected by spells and items.l such as underwater breathing

It is GOING to be affecyef by divine shield which protects you against all spells regardless of if they do or dont do damage.

**Comment by Mostprovidedlinks** [comment]
Yes to all spells regardless of if drowning is damage related or not.

Yeah lalala lets disregard all of the information from wowhead.
Prevents all damage fron spells yet drowning can bypass. Do you have any idea how stupid this sounds?

Drowning is an environmental spell affected by spells and items.l such as underwater breathing

It is GOING to be affecyef by divine shield which protects you against all spells regardless of if they do or dont do damage.

**Comment by Mostprovidedlinks** [comment]
Yes to all spells regardless of if drowning is damage related or not.

Yeah lalala lets disregard all of the information from wowhead.
Prevents all damage fron spells yet drowning can bypass. Do you have any idea how stupid this sounds?

Drowning is an environmental spell affected by spells and items.l such as underwater breathing


Yeah lala lets just disregard all the proof provided by wowhead showing it prevents all spells and not just schools. Do you have any idea how stupid an immune to all spells not working on drowning looks?
It is GOING to be affected by divine shield which protects you against all spells regardless of if they do or dont do damage.

**Comment by Mostprovidedlinks** [comment]
Yes to all spells regardless of if drowning is damage related or not.

Yeah lalala lets disregard all of the information from wowhead.
Prevents all damage from spells yet drowning can bypass. Do you have any idea how stupid this sounds?

Drowning regardless of what it is mate is affected by spells and items such as underwater breathing spells on armor or from items. Preventing drowning as mentioned means it will prevent all spells and do thesame thing as these items. It is GOING to be affected by divine shield which protects you against all spells regardless of if they do or dont do damage.

**Comment by Mostprovidedlinks** [comment]
If a spell creates an aura to make you breathe underwater forever it is still a spell.

**Comment by Mostprovidedlinks** [comment]
It doesnt need to have a school. It says all spells.

**Comment by Mostprovidedlinks** [comment]
The reason blessing of protection doesnt protect you against drowning is because it doesnt protect against

All spells.

Read wowheads changelog please and see that the vanilla divine shield is identical to the one mentioned to prevent drowning. Thanks

This seriously needs a fix

**Comment by Mostprovidedlinks** [comment]
I find it odd that someone would write thesame thing foor both iceblock and divine shield.

It can only mean it did happen.

**Comment by Mostprovidedlinks** [comment]
There is more proof stating yes than no. Where is the proof showing it never did? I dont see it.

**Comment by Mostprovidedlinks** [comment]
Effect #3 | Apply Aura: Immunity (All)


End of thread

**Comment by Nelythia** [comment]
I had a paladin as main in retail vanilla and while Divine Shield protected you from fall damage, it did not from drowning.

Just wanted to mention it, I am not too concerned about any change so I won't bother looking for any evidence.

**Comment by Mostprovidedlinks** [comment]
I played retail 2006 on Firetree as well. And I can confirm it did.

**Comment by Mostprovidedlinks** [comment]
And I dont see how this will harm anything.

**Comment by gearselysium** [comment]
> Vanilla divine shield is identical to what it does as shown on wowhead with the newer divine shield.
> Read the auras. Environmental or not, Drowning is affected BY SPELLS. Its affected by spells which give effects against it so that proves divine shield should be able to affect it.

Should BE ABLE to affect it, yes, but that doesn't mean it DOES affect it.

> Effect #3 | Apply Aura: Immunity (All)
> 
> Two pages state the divine shield on wowhead prevents drown damage and another page states iceblock prevents drown damage.

This has nothing to do with drowning. This simply sets immunity to damage schools. There are other effects that do this that have no mention of preventing drowning.

**Comment by sickheals** [comment]
I wanted to comment @gearselysium you have the patience of a saint. @Mostprovidedlinks keeps missing basic logic and leaps to conclusions.

I searched for video evidence of a vanilla paladin in water using their bubble but I couldn't find one. 

First @mostprovidedlinks you keep missing the basic understanding that drowning is not a spell. It is not physical. If you were out of breath and drowning and became immune to someone punching you, you would still be out of breath. Divine shield does not apply underwater breathing and does not afford you an additional protection against drowning. It makes you immune to physical and spell damage.

Second you keep saying that the old divine shield is identical to the new divine shield, but why did they change it? Why wouldn't Blizzard keep the same spell if they are identical? Also you assume that the auras that they apply did not change. It's likely that #37 or #127 changed even if divine shield didn't. 

Third, patch notes don't cover all changes. Many things in wow we're changed and patch notes didn't get entered for those changes. And your wowhead Wikipedia links only show major changes to the spell and are not an exhaustive search for all changes according to the patch notes. You'd have to collect all the patch notes yourself and search for drowning or divine shield in all of the patch notes to get an exhaustive search, but even then it might not be included since Blizzard did not include every change in patch notes.

**Comment by Mostprovidedlinks** [comment]
Divine Shield prevents all damage, including (unlike most other bubbles of other classes) falling damage, and even drowning damage.

Doesnt agree with you

**Comment by Mostprovidedlinks** [comment]
Ice Block can be used to avoid fall damage (see  [Blink] and  [Slow Fall] for other ways). While Ice Blocked your character falls normally, but doesn't take any damage when hitting the ground. (In previous patches, an Ice Blocked mage would hover in the air, then begin to fall when the spell expired.)

Ice Block will prevent drowning and fatigue damage, and reset the damage counter (not the bar) to zero. You will still lose breath and become fatigued while Ice Blocked.


My patience is thinning.

**Comment by Mostprovidedlinks** [comment]
Drownings damage counter is reset to zero.
Same for divine shield. It will do whatever prevents drowning just like the other items.

**Comment by Mostprovidedlinks** [comment]
Of course it provides you with the protection. If these items can do it. A spell like divine shield which provides a much more powerful spell would

**Comment by Mostprovidedlinks** [comment]
Fall damage and drown damage are both environmental damage. So they would both be affected

**Comment by Mostprovidedlinks** [comment]
You already proved me right when you said it doesnt prevent environmental damage.
Falling is environmental damage and it prevents it.

@thepparker fix this please It says it word for word.

Proof outweighs your assumptions.

Berserker is a negative buff on the server yet should never have dropped divine shield.

There is something wrong with the servers database. Berserker should not be a negative that is removed by divine shield so how can you believe already inorrect information and use that as your proof.

The fact is there is an iceblock page stating thesame thing as divine shield. That is far too much proof to toss aside which I dont find a coincidence.

**Comment by sickheals** [comment]
@mostprovidedlinks how short your memory is. Can you scroll up? He said fall damage is physical damage. Can you look through the history of that wiki page and see if the UNOFFICIAL TIPS AND TRICKS also mentioned something about falling or drowning in 2005 or 2006? 

The only evidence you've given is for someone who is making a patch 7.0+ server. That's not elysium.

**Comment by Mostprovidedlinks** [comment]
Falling damage is still Environmental damage. Research it.

**Comment by Mostprovidedlinks** [comment]
You base your reply around environmental damage being unaffected.
I have proven this false.

**Comment by Mostprovidedlinks** [comment]
And why would it say in older versions hence hence then mention drowning 

And why would it say it about iceblock and dovine shield

And why would it mention environmental damages next to each other.

**Comment by Mostprovidedlinks** [comment]
You fail to understand this.

Divine Shield prevents all damage, including (unlike most other bubbles of other classes)

**Comment by Mostprovidedlinks** [comment]
Then you arent reading the changelog correctly. Go to the version of divine shield I have stated above. Its the rank 2 shield that is the exact version of vanilla. And see how it is identical it is. And that page shows an cataclysm divine shield. Not a 7.0. You obviously have no clue.

**Comment by sickheals** [comment]
If what you say is true, you should be able to find evidence of this from that period. All your evidence is circumstantial and is based off incorrect assumptions.

Now if you contend that fall damage is not physical, that should probably be a separate ticket with its own evidence.

I did find the date for when the UNOFFICIAL information about drowning was added in 2007 after TBC. wowwiki.wikia.com/wiki/Divine_Shield?diff=next&oldid=901159

**Comment by Mostprovidedlinks** [comment]
You obviously have not read the comments patch dates telling us what version the gamepedia divine shield is here

http://www.wowhead.com/spell=642/divine-shield#comments

**Comment by Mostprovidedlinks** [comment]
You obviously dont realise how this will not hurt any class when its fixed.

**Comment by sickheals** [comment]
If they are exactly the same spell, why did Blizzard change and use a different spell?

**Comment by Mostprovidedlinks** [comment]
You do realise the bubble from tbc and vanilla is identical.
Thanks for proving me right again.

**Comment by sickheals** [comment]
They apply the same auras, but did the auras change? Why did Blizzard use a different spell id for the new divine shield? You keep avoiding the most basic question.

**Comment by Mostprovidedlinks** [comment]
They have identical spell ids.

**Comment by Mostprovidedlinks** [comment]
https://tbc-twinhead.twinstar.cz/?spell=1020

**Comment by Mostprovidedlinks** [comment]
https://tbc-twinhead.twinstar.cz/?spell=1020

**Comment by Mostprovidedlinks** [comment]
http://db.vanillagaming.org/?spell=1020

**Comment by Mostprovidedlinks** [comment]
https://tbc-twinhead.twinstar.cz/?spell=1020

As I said they are identical. 

This will be fixed.

**Comment by Mostprovidedlinks** [comment]
Youve already proven to me tbc divine shields version prevents drowning. Thanks.

And since both vanilla ans tbc are identical. Thats all we need.

**Comment by Mostprovidedlinks** [comment]
Youve already proven to me tbc divine shields version prevents drowning. Thanks.

And since both vanilla and tbc are identical. Thats all we need.

**Comment by sickheals** [comment]
You have a crappy attitude that makes people not want to help you. Thanks.

**Comment by Mostprovidedlinks** [comment]
Maybe because its 4 am and people need to sleep.

**Comment by Xaph-er** [comment]
This is the most epic bug thread that amounts to absolutely no real impact on the actual game. Why do oeople hate paladins so much?

**Comment by Mostprovidedlinks** [comment]
How much proof do they want? seriously

**Comment by Mostprovidedlinks** [comment]
Sick the effects are identical and both use

Effect #1 Apply Aura: Immunity (Physical)
Effect #2 Apply Aura: Mod Melee Attack Speed - %
Value: -100%
Effect #3 Apply Aura: Immunity (All)
Flags
Remove auras on immunity

**Comment by Mostprovidedlinks** [comment]
@guybrushgit any news on when this will be getting Fixed?
